### ISSUE TITLE
anjay_ongoing_registration_exists() remains true when a bootstrapping is used.

### ISSUE THREAD

**Issue by IsaacDynamo** [AVSystem/Anjay#56]
Expected `anjay_ongoing_registration_exists()` to return `false` when bootstrapping is done and client registered to the lwm2m server.

### Testcase
Run the demo with a bootstrap server. After the client is connected to the lwm2m server, `anjay_ongoing_registration_exists()` returns `true`.
```none
output/bin/demo -e test_anjay_ongoing_registration_exists -b -u coap://leshan.eclipseprojects.io:5783
(DEMO)>ongoing-registration-exists
ONGOING_REGISTRATION==true
```

When a lwm2m server is used directly without bootstrapping, `anjay_ongoing_registration_exists()` will return `false` as expected when the client is connected.
```none
output/bin/demo -e test_anjay_ongoing_registration_exists -u coap://leshan.eclipseprojects.io:5683
(DEMO)>ongoing-registration-exists
ONGOING_REGISTRATION==false
```

### Version
Anjay 2.13.0

**Comment by Mierunski** [comment]
Hi!
Thank you for the issue, we added it to our tracking system and will work on the solution

**Comment by kFYatek** [comment]
Hi,
This issue should be fixed in the 2.14.0 release that has just been pushed.
